src/vivarium/framework/results/interface.py

```diff
@@ -143,4 +143,42 @@ def register_observation(
         excluded_stratifications: List[str] = (),
         when: str = "collect_metrics",
```

Comment:
The docs mention this could be a list. Would there ever be a desire to register the same stratifications at different times?